packages/apps/marmalade-marketplace/src/pages/tokens/[tokenId]/index.tsx

```diff
@@ -0,0 +1,247 @@
+                </TabItem>
+                <TabItem title="Sale">
+                  <CrudCard title="Make a Sale" description={[]}>
+                    <div className={styles.formContainer}>
+                      <Select onSelectionChange={onSaleTypeChange} label="Select a sale contract">
+                        <SelectItem key="none">None</SelectItem>
+                        <SelectItem key="conventional">Conventional Auction</SelectItem>
+                        <SelectItem key="dutch">Dutch Auction</SelectItem>
+                      </Select>
+                        <NumberField value={saleData.amount} onValueChange={(value:number) => {onSaleDataChange('amount', value)}} label="Amount" minValue={0.1} placeholder="Set the amount to sell" startVisual={<KRoundedFilledKdacolorBlack />} />
+                      <NumberField value={saleData.price} onValueChange={(value:number) => {onSaleDataChange('price', value)}}label="Price" minValue={0.1} placeholder="Set the token price in KDA" />
+                      <NumberField value={saleData.timeout} onValueChange={(value:number) => {onSaleDataChange('timeout', value)}} label="Timeout" minValue={1} startVisual={<MonoAccessTime />} description="Set the minumum amount of days that the sale will be valid" />
+                    </div>
+                  </CrudCard>
+                  </TabItem>
+              </Tabs>
+              {saleData.saleType === "conventional" && (
+              <CrudCard
+                title="Conventional Auction"
+                description={[
+                  "Allow bidding on the token up untill a certain time",
+                  "The highest bidder at the end of the auction wins the token",
+                  "The seller can choose to set a reserve price",
+                ]}
+              >
+                <div>
+                  <p>Form to setup conventional auction</p>
+                </div>
+              </CrudCard>)}
+            {saleData.saleType === "dutch" && (
+              <CrudCard
+                title="Dutch Auction"
+                description={[
+                  "The price of the token is set high and decreases over time",
+                  "The first bidder to accept the price wins the token",
+                  "The seller can choose to set a start and reserve price"
+                ]}
+              >
+                <div>
+                  <p>Form to setup dutch auction</p>
+                </div>
+              </CrudCard>)}
+            
+          </Stack>
+          <Stack className={styles.buttonRowContainer}>
+            <Button variant="outlined" onPress={onCancelPress}>
+              Cancel
+            </Button>
+            <Button onPress={onCreateSalePress} isDisabled={!isSaleValid()}>
+              Create Sale
+            </Button>
+          </Stack>
+        </div>) 
+        : (
+          <SendTransaction send={send} preview={preview} poll={poll} transaction={transaction}  />
```

Comment:
Same comment on the transaction component, since we have a single transaction page